### ISSUE TITLE
Local LLM  pandasai.json 

### ISSUE THREAD

**Issue by ahadda5** [Sinaptik-AI/pandas-ai#1488]
### System Info

  "name": "pandasai-all",
  "version": "1.0.0",
MacOS (15.1.1) 

The code is run directly as Poetry run. and not in a docker. 

### 🐛 Describe the bug

I'm trying to use a local LLM but it keeps defaulting to BamboLLM
Here how the pandasai.json  at the root directory looks like

Regardless what "llm" variable i use  it  defaults to BambooLLM!     
```
  "llm": "LLM",
  "llm_options": {
    "model": "Llama-3.3-70B-Instruct",
    "api_url": "http://localhost:9000/v1"
  }
```

This is the supported list 
```
__all__ = [
    "LLM",
    "BambooLLM",
    "AzureOpenAI",
    "OpenAI",
    "GooglePalm",
    "GoogleVertexAI",
    "GoogleGemini",
    "HuggingFaceTextGen",
    "LangchainLLM",
    "BedrockClaude",
    "IBMwatsonx",
]
```

**Comment by ahadda5** [comment]
@dosu  yes LLM is found in the `pandasai.json` and the supported list. 
Talk to me about the override_config in the file you mentioned  `pandasai/config.py ` it seems the bug is that its defaulting to llm.BambooLLM regardless what is the "llm" variable is.

**Comment by ahadda5** [comment]
Its a bug. Here are the highlights

-  `pandasai/llm/__init__.py` needs to export `localLLM `
- `pandasai/schemas/df_config.py`  needs to have  `not isinstance(llm, (LLM, LocalLLM, LangChainLLM)` to avoid returning it as `BambooLLM`

**Comment by gdcsinaptik** [comment]
Hey @ahadda5 are you using the latest version of PandasAI (2.4.1)?

**Comment by ahadda5** [comment]
i pulled the main branch yesterday.  not sure what version.

**Comment by ahadda5** [comment]
so my pandasai.json was not found which definitly didn't help.  The working directory was in ./server and couldn't find the pandasai.json at the root level. 
I changed the config file to return localLLM and i can now connect .
